use_cases/2024_07_HTGR-TES_Technoeconomic Case Study/ARMA/README.rtf

```diff
@@ -0,0 +1,8 @@
+{\rtf1\ansi\ansicpg1252\cocoartf2709
+\cocoatextscaling0\cocoaplatform0{\fonttbl\f0\fswiss\fcharset0 Helvetica;}
+{\colortbl;\red255\green255\blue255;}
+{\*\expandedcolortbl;;}
+\margl1440\margr1440\vieww11520\viewh8400\viewkind0
+\pard\tx720\tx1440\tx2160\tx2880\tx3600\tx4320\tx5040\tx5760\tx6480\tx7200\tx7920\tx8640\pardirnatural\partightenfactor0
+
+\f0\fs24 \cf0 This folder contains the trained ARMA files used in the M2 simulation. No changes have been made to the ARMA input files; they are essentially identical to the files available in the shared folder.}
```

Comment:
Could you also explain the different data data sets and abbreviations you use in here, as well as what variables/years were trained with each ARMA set?